canvas_modules/common-canvas/src/common-properties/components/control-item/control-item.jsx

```diff
@@ -46,12 +46,15 @@ class ControlItem extends React.Component {
 			let tooltip;
 			if (this.props.control.description && !isEmpty(this.props.control.description.text)) {
 				if (this.props.control.description.placement === "on_panel") {
-					description = <div className="properties-control-description">{this.props.control.description.text}</div>;
+					description = <div className="properties-control-description">{this.props.control.description.text}</div>; // add link here???
```

Comment:
When description has placement `on_panel`, should we display link here?

Reply:
For now we can leave it off.